.github/workflows/virtualbox_image.yml

```diff
@@ -0,0 +1,26 @@
+name: Create VirtualBox disk image from Dockerfile
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  push:
+    paths:
+      - 'docker/*'
+      - '.github/workflows/virtualbox_image.yml'
+
+env:
+  VM_PASSWORD: temppwd
+
+jobs:
+  virtualbox_image_build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Build VirtualBox image using d2vm
+        run: |
+          curl -sL "https://github.com/linka-cloud/d2vm/releases/download/v0.2.0/d2vm_v0.2.0_linux_amd64.tar.gz" | tar -xvz d2vm
+          sudo mv d2vm /usr/local/bin/
+          echo "make worked"
+          sudo d2vm build -p ${{ env.VM_PASSWORD }} -f ./docker/ubuntu_22.04-dev.dockerfile -o pyne.vdi .
```

Comment:
Once this is working, I think you'll need to do something in github actions to capture the file that produced as an asset.